RiverLoggerApi/Models/AuthenticateRequest.cs

```diff
@@ -2,7 +2,7 @@
 {
     public class AuthenticateRequest
     {
-        public required string Username { get; set; }
+        public required string Email { get; set; }
```

Comment:
Maybe we could use username and email auth ?

Reply:
like have them both?

Reply:
we will see when we come to that for now we will use only email